lib/scoring.rb

```diff
@@ -1,9 +1,61 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'player'
+
 module Scrabble
   class Scoring
+    attr_reader :word, :letter
```

Comment:
Why have an `attr_reader` here?  You will **never** create an instance of `Scoring`  It only has class methods!